.github/workflows/release.yml

```diff
@@ -0,0 +1,30 @@
+name: Release
+
+on:
+  release: { types: [published] }
+  workflow_dispatch:
+
+jobs:
+  npmjs:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v2
+        with:
+          registry-url: "https://registry.npmjs.org"
+      - run: npm publish --ignore-scripts
+        env:
+          NODE_AUTH_TOKEN: ${{ secrets.NPM_TOKEN }}
+
+  github-npm:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v2
+        with:
+          registry-url: "https://npm.pkg.github.com"
+      - name: scope package name as required by GHPR
+        run: npm init -y --scope ${{ github.repository_owner }}
```

Comment:
I have no npm experience, why is this necessary here, but not in the other job?

Reply:
On npm, the package is published unscoped (as `bats`). The GHPR, on the other hand, doesn't support or allow unscoped packages. The package name, therefore, must be scoped to the organization: `@bats-core/bats`.

This was the simplest way I could think of to change the package.json such that the package name is scoped prior to publishing to GHPR.

An alternative would be to deprecate the unscoped `bats` package and publish it scoped on both registries. That might be something we should consider.

Reply:
I'd say we can defer this discussion to get v1.3.0 out the door. Maybe you want to open an issue for that?